server/src/internalClusterTest/java/org/opensearch/search/aggregations/metrics/SumLongIT.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
```

Comment:
@matthewryanwells could you please remove `Elasticsearch` related headers? This is new file, 100% under OpenSearch governance

Reply:
Just to confirm, all files already created and just edited should have the `Elasticsearch` header, while all new files just need this apache header?:
```/*
 * SPDX-License-Identifier: Apache-2.0
 *
 * The OpenSearch Contributors require contributions made to
 * this file be licensed under the Apache-2.0 license or a
 * compatible open source license.
 */```

Reply:
removed in https://github.com/opensearch-project/OpenSearch/pull/6984/commits/7d50f500d045f1c8acb3d71318ef777cef6e6143